Doc/library/socket.rst

```diff
@@ -149,6 +149,14 @@ created.  Socket addresses are represented as follows:
 
   .. versionadded:: 3.6
 
+- :const:`AF_VSOCK` allows communication between virtual machines and
+   their hosts. The sockets are represented as a (CID, port) tuple
+   where the context ID or CID and port are integers.
+
+  Availability: Linux >= 4.8. QEMU >= 2.8.
```

Comment:
Well I thought it would be better informationally to have it in but if it is not acceptable I can remove it.